x-pack/plugins/enterprise_search/public/applications/app_search/components/result/result.scss

```
@@ -0,0 +1,55 @@
+.appSearchResult {
+  overflow: hidden;
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+
+  &__contentWrap {
```

Follow-up on my previous comment/suggestion on renaming `__contentInner` to `__body` - I think we can safely make this just `__content` w/o confusion which is a bit simpler

```suggestion
  &__content {
```